include/boost/gil/image.hpp

```diff
@@ -37,7 +38,7 @@ namespace boost { namespace gil {
 ///
 ////////////////////////////////////////////////////////////////////////////////////////
 
-template< typename Pixel, bool IsPlanar = false, typename Alloc=std::allocator<unsigned char> >
+template< typename Pixel, bool IsPlanar, typename Alloc>
```

Comment:
This this seems like opposite of the change in the `include/boost/gil/metafunctions.hpp`:

```
- template <typename Pixel, bool IsPlanar, typename Alloc> class image;
+ template <typename Pixel, bool IsPlanar = false, typename Alloc=std::allocator<unsigned char> > class image;
```

Intentional?

Reply:
yes, the build was failing after moving things, so I was forced to do this.